dockers/docker-base-bullseye/Dockerfile.j2

```diff
@@ -83,17 +83,23 @@ RUN pip3 install supervisord-dependent-startup==1.4.0
 
 RUN mkdir -p /var/log/supervisor /etc/supervisor/conf.d
 
-# Install gcc, libc6-dev and python3-dev for compiling python-lzf
-RUN apt-get -y install build-essential libc6-dev python3-dev
+# Install gcc, libc6-dev and git for compiling librdb
+RUN apt-get -y install build-essential libc6-dev git
 
-# Install python-lzf
-RUN pip3 install 'python-lzf==0.2.4'
+# Obtain librdb source codes
```

Comment:
>codes [](http://example.com/codeflow?start=23&length=5)

The original design of this docker-base image is just the most common base docker image which all other docker image could be based on. It should only download and install vanilla debian packages or well-known pypi packages. Please move other build process for packages like `python-lzf` or `librdb` into dedicated build target, and only install them in requiring docker images.

Reply:
I have a solution below:
1.  Add a new folder librdb into sonic-sairedis https://github.com/sonic-net/sonic-sairedis
2.  In the folder librdb, get source codes by: 	git clone https://github.com/redis/librdb.git
3.  Compile the librdb and install the rdb-cli into syncd container.
@kcudnik, @qiluo-msft, do you agree with me? Thanks.

Reply:
[Update] Hi @qiluo-msft , @kcudnik disagrees with the above solution. I move it into the below file shown below. Please review, thanks.
https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/docker-syncd-brcm-dnx/Dockerfile.j2